sqlparse/filters.py

```diff
@@ -561,8 +563,231 @@ def process(self, stack, stream):
                     mode = 1
 
 
+ColumnAttributeProperty = namedtuple('ColumnAttributeProperty', 'has_value is_type_attribute')
+
+
+class MysqlCreateStatementFilter(object):
+
+    @classmethod
+    def process(cls, stack, statement):
+        if statement.get_type() != 'CREATE':
+            return
+        tokens = []
+        tokens.append(statement.token_first())
+        table_name = cls._get_table_name(statement)
+        tokens.append(sql.TableName(value=table_name, ttype=T.Name))
+        tokens.append(cls._get_columns(statement))
+        statement.tokens = tokens
+        return
+
+    @classmethod
+    def _get_table_name(cls, statement):
+        # 1st Name type is the table name
+        table_name_token = statement.token_next_by_type(0, T.Name)
+        if not table_name_token:
+            raise SQLParseError('Cannot find table name.')
+        return cls._clean_quote(table_name_token.value)
+
+
+    @classmethod
+    def _clean_quote(cls, text):
+        return text.strip('"`\'')
+
+    @classmethod
+    def _get_columns(cls, statement):
+        # Get the Parenthesis which contains column definitions
+        parenthesis_token = statement.token_next_by_instance(0, sql.Parenthesis)
+        if not parenthesis_token:
+            raise SQLParseError('Cannot find column definitions')
```

Comment:
I'm not entirely sure if we'd like to start doing this since this is a no-validation parser. If it cannot find any column definition, then it'll have empty list of columns.